src/arch/host/config.h.in

```diff
@@ -1,2 +1,4 @@
 #define CONFIG_HOST @CONFIG_HOST@
 #define CONFIG_LIB @CONFIG_LIB@
+#define CONFIG_COMP_VOLUME
+#define CONFIG_COMP_SRC
```

Comment:
Should be `#define CONFIG_COMP_SRC @CONFIG_COMP_SRC@`
You also need SETs in src/arch/host/CMakeLists

Host builds everything as shared libs

Reply:
Fixed, thanks